src/components/InputField.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+export default function InputField({ label, ...inputProps }) {
```

Comment:
Personal preference: I think `labelText` would be a better name.
```suggestion
export default function InputField({ labelText, ...inputProps }) {
```